coding/review.md

```diff
@@ -0,0 +1,14 @@
+# Code Review
+
+## Suggestions for Effective Review
+
+[tbd]
+
+## Etiquette
+
+### Approval with suggestions for trivial changes
+In some circumstances, if a branch requires a small number of trivial changes you may approve the branch with the caveat that issues you've identified are resolved or considered, rather than marking the branch *needs fixing* (lp) or *request changes* (gh). These changes should be in the category of minor adjustments to file structure, fixing typos, renamings, small css tweaks etc.
```

Comment:
Would it be okay to expand on `In some circumstances` a bit? What circumstances?

How about:

> Reviewers may consider approving a branch with the caveat that issues they've identified are resolved or considered, rather than marking the branch [*needs fixing* on Launchpad](https://help.launchpad.net/Code/Review#The_flow_of_this_process) or [*request changes* on GitHub](https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/#submitting-your-review).
> 
> They may only do this when a branch requires a small number of trivial changes (e.g. moving files, fixing typos, renaming variables, small CSS tweaks), and they have reason to expect that they may block progress by not being available for a second review – e.g. because of different timezones.
> 
> It is the branch author's responsibility to ensure they have addressed these suggestions before merging their branch. They still should seek an additional review following their changes wherever possible.

Reply:
Yep, I think that reads well, thanks Robin. Have updated the copy.